client/post-editor/editor-revisions-list/item.jsx

```diff
@@ -37,30 +37,30 @@ class EditorRevisionsListItem extends PureComponent {
 						} ) }
 				</span>
 				<div className="editor-revisions-list__changes">
-					{ this.props.revision.changes.added > 0 && (
+					{ this.props.revision.summary.added > 0 && (
```

Comment:
Just a thought here and a little outside the scope of this PR - would it be worth creating variables like so to reduce some noise / improve readibility: 
```js
const hasAddedChanges = this.props.revision.summary.added > 0;
const hasRemovedChanges = this.props.revision.summary.removed > 0;
// ...
{ hasAddedChanges && hasRemovedChanges && ', ' }
```
We could go one step further and use `get()` to as, as it stands, we're accessing chained properties that aren't reflected in the proptypes.